airbyte-webapp/src/pages/OnboardingPage/components/DestinationStep.tsx

```diff
@@ -97,6 +87,7 @@ const DestinationStep: React.FC<Props> = ({ onNextStep, onSuccess }) => {
       errorMessage={errorMessage}
       selectedConnectorDefinitionSpecification={destinationDefinitionSpecification}
       isLoading={isLoading}
+      formValues={destinationDefinitionId ? { serviceType: destinationDefinitionId } : undefined}
```

Comment:
Figuring out, if can we do it easily...
Since some components use it also

<img width="496" alt="image" src="https://user-images.githubusercontent.com/20929344/195356905-43b6935d-7276-4200-9a94-54841444e856.png">